lms/events/event.py

```diff
@@ -101,3 +102,33 @@ def _get_assignment_id(self):
             return assignment.id
 
         return None
+
+
+class AdminChangeEvent(BaseEvent):
+    def __init__(self, instance, *args, **kwargs):
+        self._instance = instance
+        super().__init__(*args, **kwargs)
+
+    def _get_data(self):
+        changes = {}
```

Comment:
TODO this is a bit opaque. Add some comments here.

Reply:
I remember, this works because all `_get_<field>` methods cause something to happen right?